_docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md

```diff
@@ -401,12 +380,12 @@ data so you can apply even deeper SQL functionality. Here is a sample query:
     | Fashion          | 1897        |
     |------------------|-------------|
 
-Stay tuned for more features and upcoming activities in the Drill community.
+与 Drill 社区保持密切联系来获得更多的特性以及了解即将到来的活动。
 
-To learn more about Drill, please refer to the following resources:
+想更多了解 Drill，请参考如下资源：
 
-  * Download Drill here: <http://getdrill.org/drill/download>
-  * [10 reasons we think Drill is cool]({{site.baseurl}}/docs/why-drill)
-  * [A simple 10-minute tutorial]({{ site.baseurl }}/docs/drill-in-10-minutes>)
-  * [More tutorials]({{ site.baseurl }}/docs/tutorials-introduction/)
+  * 下载 Drill: <http://getdrill.org/drill/download>
```

Comment:
Replace the download link to official. Please sync to the english docs.

Reply:
resolved